api/webhook-vs-polling.md

```diff
@@ -146,18 +145,26 @@ application/json
 [Search orders OpenAPI Specification](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order-search/specs.yaml#/order-search/searchRoute)
 {% endhint %}
 
+{% hint style="info" %}
+[Search shipments OpenAPI Specification](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/shipment/specs.yaml#/shipment/searchShipmentsRoute)
+{% endhint %}
+
 #### Step 2. Process the orders in the search response body
 
-See the [Search orders OpenAPI Specification](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order-search/specs.yaml#/order-search/searchRoute) and [Receive order response](./) for order fields descriptions.
+See the [Search orders OpenAPI Specification](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order-search/specs.yaml#/order-search/searchRoute).
+
+* Use the `lastUpdatedAt` on order line level to filter on if the line has been changed.
+* Use the `status` field to filter on order process and logistics status.
+
+See the [Search shipments OpenAPI Specification](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/shipment/specs.yaml#/shipment/searchShipmentsRoute).
```

Comment:
Should this be 2 separate blocks? It feels a bit weird.
Perhaps link both API specs, and then provide intructions (instructions are the same, right?)

Reply:
Removed the embedded, duplicate API spec, it was something from the past, we now have real API specs